### PR TITLE
Search for class

### PR DESCRIPTION
**Pull request by aaronlhh**
Adding searchClass page with backend files and css files: 
 
Did not test the backend files yet, but want to push to main and get a first look. 
Front end works great. The buttons will not have any functions until the backend is finished

### CODE REVIEW

**Review by anhhoang1402 — APPROVED** [review]
Nice work!

File: portfolio/src/main/webapp/searchClass.html [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+    <!-- pending: display the search result -->
+    <section class="result-table" >
+        <table>
+            <thead>
+                <th>College/University</th>
+                <th>Department</th>
+                <th>Class Info</th>
+                <th>Semester</th>
+                <th>Professor</th>
```
Comment (anhhoang1402) [inline portfolio/src/main/webapp/searchClass.html:148]: The front-end looks amazing! Just one thing, can you add more space among the column headers? They look a bit condensed here.

image

**Review by anhhoang1402 — COMMENTED** [review]
Amazing work on both the front-end and back-end!

File: portfolio/src/main/webapp/searchClass.html (hunk 1 above)
Comment (aaronlhh) [reply portfolio/src/main/webapp/searchClass.html:148]: you are right! I was using a monitor so didnt count the scenario of laptops! I will fix it later!

**Pull request merged**